packetbeat/protos/nfs/nfs4.go

```diff
@@ -167,18 +167,20 @@ func (nfs *Nfs) eatData(op int, xdr *Xdr) {
 
 func (nfs *Nfs) getV4Opcode(xdr *Xdr) string {
```

Comment:
we do return last seen on purpose. Here it the why. Compound request group multiple nfs operations into a sine request. For example:

PUTFH + READDIR + GETATTR

can be part of a single request. In this example READDIR is the main operation, triggered by **ls** command.  On an other hand in:

PUTFH + GETATTR

GETATTR is the main operation, triggered by stat call.

The whole 'main operation' idea is based on fact, that clients more or less combine first-class operations, like OPEN, WRITE, READDIR with some less important opertions - GETATTR, PUTFH and so on. Nevertheless, 'second class' operations can by them self be a 'main' operation in a different context. The function getV4Opcode tries to guess main purpose of the request. I will think how to restructure it to make it more obvious.